src/lib/y2network/config_reader.rb

```diff
@@ -28,9 +28,9 @@ module ConfigReader
     # @param opts   [Hash] Reader options
     # @return [#config] Configuration reader from {Y2Network::ConfigReader}
```

Comment:
```suggestion
     # @return [Y2Network::Autoinst::ConfigReader,Y2Network::Sysconfig::ConfigReader]
```
also the class comment above is outdated.